runtime/queries/smali/highlights.scm

```diff
@@ -0,0 +1,218 @@
+; Types
+
+(class_identifier
+  (identifier) @type)
+
+(primitive_type) @type.builtin
+
+((class_identifier
+   . (identifier) @_first @type.builtin
+   (identifier) @type.builtin)
+  (#any-of? @_first "android" "dalvik" "java" "kotlinx"))
+
+((class_identifier
+   . (identifier) @_first @type.builtin
+   .  (identifier) @_second @type.builtin
+   (identifier) @type.builtin)
+  (#eq? @_first "com")
+  (#any-of? @_second "android" "google"))
+
+; Methods
+
+(method_definition
+  (method_signature (method_identifier) @method))
```

Comment:
Quite a few scopes in this file don't match our expected scopes so they won't highlight correctly: https://docs.helix-editor.com/themes.html#syntax-highlighting

Reply:
Ok I'll work on it!

I just copied these scm files from upstream and it looked good enough to me in the smali files I am editing

Reply:
Should be much better now! I left my changes as a separate commit for review, but I can easily rebase this PR into one commit if you prefer